<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>178</width>
     <height>174</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="bpm_grid">
    <item row="1" column="2">
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="bpm_name">
      <property name="text">
       <string>BPM N</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="CXSpinBox" name="bpm_delay">
      <property name="minimum">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Delay</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="1" colspan="3">
     <widget class="Line" name="line_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="CXSpinBox" name="bpm_numpts">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>1024</number>
      </property>
      <property name="cname" stdset="0">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="5" rowspan="6">
     <widget class="Line" name="line_16">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="Line" name="line_15">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="CXSpinBox" name="bpm_ptsofs">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="cname" stdset="0">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>NumPts</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>PtsOfs</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="2" rowspan="3">
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="6">
     <widget class="Line" name="line_17">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="CXSwitch" name="bpm_is_on"/>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <layout class="QVBoxLayout" name="cbox_grid"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CXSpinBox</class>
   <extends>PSpinBox</extends>
   <header>cxwidgets</header>
  </customwidget>
  <customwidget>
   <class>CXSwitch</class>
   <extends>QPushButton</extends>
   <header>cxwidgets</header>
  </customwidget>
  <customwidget>
   <class>PSpinBox</class>
   <extends>QSpinBox</extends>
   <header>cxwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
